apps/condo/domains/organization/schema/OrganizationEmployeeRole.js

```diff
@@ -68,6 +68,8 @@ const OrganizationEmployeeRole = new GQLListSchema('OrganizationEmployeeRole', {
         canManageTicketComments: { type: Checkbox, defaultValue: true },
         canManageDivisions: { type: Checkbox, defaultValue: false },
         canShareTickets: { type: Checkbox, defaultValue: true },
+        canManageBillingReceipts: { type: Checkbox, defaultValue: false },
```

Comment:
Only integration can manage billing receipts! You can't manage it by anyone else